Description: Rename manpages to kde5 compatibility. 
 Manpages conflict with kde4 installed manpages. Renamed to version 5.
 .
 kdoctools (4.95.0-0ubuntu1~ubuntu14.04~ppa4) trusty; urgency=low
 .
   * Initial release
Author: Scarlett Clark <scarlett@scarlettgatelyclark.com>

---

--- kdoctools-4.95.0.orig/docs/CMakeLists.txt
+++ kdoctools-4.95.0/docs/CMakeLists.txt
@@ -1,5 +1,5 @@
-add_subdirectory(checkXML)
+add_subdirectory(checkXML5)
 add_subdirectory(meinproc5)
 
-add_subdirectory(kdeoptions)
-add_subdirectory(qtoptions)
+add_subdirectory(kde5options)
+add_subdirectory(qt5options)
--- /dev/null
+++ kdoctools-4.95.0/docs/checkXML5/CMakeLists.txt
@@ -0,0 +1 @@
+kdoctools_create_manpage(man-checkXML5.1.docbook 1 INSTALL_DESTINATION ${MAN_INSTALL_DIR})
--- /dev/null
+++ kdoctools-4.95.0/docs/checkXML5/man-checkXML5.1.docbook
@@ -0,0 +1,43 @@
+<?xml version="1.0" ?>
+<!DOCTYPE refentry PUBLIC "-//KDE//DTD DocBook XML V4.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
+<!ENTITY % English "INCLUDE">
+]>
+
+<refentry lang="&language;">
+<refentryinfo>
+<title>&kde; User's Manual</title>
+<author><personname><firstname>Lauri</firstname><surname>Watts</surname></personname><email>lauri@kde.org</email></author>
+<date>2003-07-03</date>
+</refentryinfo>
+
+<refmeta>
+<refentrytitle><command>checkXML5</command></refentrytitle>
+<manvolnum>1</manvolnum>
+</refmeta>
+
+<refnamediv>
+<refname><command>checkXML5</command></refname>
+<refpurpose>An &XML; lint tool for &kde; DocBook &XML; documents.</refpurpose>
+</refnamediv>
+
+<refsynopsisdiv>
+<cmdsynopsis>
+<command>checkXML5</command> <arg><replaceable>filename</replaceable></arg>
+</cmdsynopsis>
+</refsynopsisdiv>
+
+<refsect1>
+<title>Description</title>
+
+<para><command>checkXML5</command> is a tool to check for syntax errors in &kde;
+DocBook &XML; files.  It can also be used for other DocBook based
+&XML; files, but you should use the less specific xmllint(1) tool if
+you are not writing or otherwise working on &kde;
+documentation.</para>
+</refsect1>
+
+<refsect1>
+<title>See Also</title>
+<para>meinproc5(1) xmllint(1)</para>
+</refsect1>
+</refentry>
--- /dev/null
+++ kdoctools-4.95.0/docs/kde5options/CMakeLists.txt
@@ -0,0 +1 @@
+kdoctools_create_manpage(man-kde5options.7.docbook 7 INSTALL_DESTINATION ${MAN_INSTALL_DIR})
--- /dev/null
+++ kdoctools-4.95.0/docs/kde5options/man-kde5options.7.docbook
@@ -0,0 +1,81 @@
+<?xml version="1.0" ?>
+<!DOCTYPE refentry PUBLIC "-//KDE//DTD DocBook XML V4.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
+<!ENTITY % English "INCLUDE">
+]>
+
+<refentry lang="&language;">
+<refentryinfo>
+<title>KDE User's Manual</title>
+<author>&Lauri.Watts; &Lauri.Watts.mail;</author>
+<date>2004-02-02</date>
+<productname>K Desktop Environment</productname>
+</refentryinfo>
+
+<refmeta>
+<refentrytitle>kde5options</refentrytitle>
+<manvolnum>7</manvolnum>
+</refmeta>
+
+<refnamediv>
+<refname>kde5options</refname>
+<refpurpose>Common commandline options for all applications based on the KDE libraries</refpurpose>
+</refnamediv>
+
+<refsynopsisdiv>
+<cmdsynopsis>
+<command><replaceable>KDE application</replaceable></command>
+<arg choice="opt">KDE Generic Options</arg>
+</cmdsynopsis>
+</refsynopsisdiv>
+
+
+<refsect1>
+<title>Options</title>
+
+<para>The following options apply to all &kde; applications:</para>
+<variablelist>
+<varlistentry>
+<term><option>--caption</option> <parameter>caption</parameter></term>
+<listitem><para>Use <parameter>caption</parameter> as name in the
+titlebar.</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>--icon</option> <parameter>icon</parameter></term>
+<listitem><para>Use <parameter>icon</parameter> as the application icon.</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>--config</option> <parameter>filename</parameter></term>
+<listitem><para>Use the alternative configuration <parameter>filename</parameter></para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>--nocrashhandler</option></term>
+<listitem><para>Disable the crash handler, to get core dumps.</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>--waitforwm</option></term>
+<listitem><para>Waits for a WM_NET compatible windowmanager.</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>--style</option> <parameter>style</parameter></term>
+<listitem><para>Sets the application &GUI; style.</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>--geometry</option> <parameter>geometry</parameter></term>
+<listitem><para>Sets the client geometry of the main widget.</para></listitem>
+</varlistentry>
+</variablelist>
+
+</refsect1>
+
+<refsect1>
+<title>See Also</title>
+<para><replaceable>qtoptions</replaceable>(7)</para>
+
+</refsect1>
+
+<refsect1>
+<title>Authors</title>
+<para>Man page written by &Lauri.Watts; &Lauri.Watts.mail;</para>
+</refsect1>
+
+</refentry>
--- /dev/null
+++ kdoctools-4.95.0/docs/qt5options/CMakeLists.txt
@@ -0,0 +1 @@
+kdoctools_create_manpage(man-qt5options.7.docbook 7 INSTALL_DESTINATION ${MAN_INSTALL_DIR})
--- /dev/null
+++ kdoctools-4.95.0/docs/qt5options/man-qt5options.7.docbook
@@ -0,0 +1,244 @@
+<?xml version="1.0" ?>
+<!DOCTYPE refentry PUBLIC "-//KDE//DTD DocBook XML V4.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
+<!ENTITY % English "INCLUDE">
+]>
+
+<refentry lang="&language;">
+<refentryinfo>
+<title>KDE User's Manual</title>
+<author>&Lauri.Watts; &Lauri.Watts.mail;</author>
+<date>2004-02-02</date>
+<productname>K Desktop Environment</productname>
+</refentryinfo>
+
+<refmeta>
+<refentrytitle>qt5options</refentrytitle>
+<manvolnum>7</manvolnum>
+</refmeta>
+
+<refnamediv>
+<refname>qt5options</refname>
+<refpurpose>Common commandline options for all applications based on the Qt toolkit</refpurpose>
+</refnamediv>
+
+<refsynopsisdiv>
+<cmdsynopsis>
+<command><replaceable>Qt application</replaceable></command>
+<arg choice="opt">Qt Generic Options</arg>
+</cmdsynopsis>
+</refsynopsisdiv>
+
+
+<refsect1>
+<title>Options</title>
+
+<para>The following options apply to all &Qt; applications:</para>
+
+<variablelist> <!-- all this could be entitiesan entity -->
+<varlistentry>
+<term>
+<option>--display</option> <parameter>displayname</parameter>
+</term>
+<listitem>
+<para>Use the X-server display
+<parameter>displayname</parameter>.</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--session</option> <parameter>sessionId</parameter>
+</term>
+<listitem>
+<para>Restore the application for the given
+<parameter>sessionId</parameter>.</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--cmap</option>
+</term>
+<listitem>
+<para>Causes the application to install a private color map on an
+8-bit display.</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--ncols</option> <parameter>count</parameter>
+</term>
+<listitem>
+<para>Limits the number of colors allocated in the color cube on an
+8-bit display, if the application is using the
+<methodname>QApplication::ManyColor</methodname> color
+specification.</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--nograb</option>
+</term>
+<listitem>
+<para>Tells &Qt; to never grab the mouse or the keyboard.</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--dograb</option>
+</term>
+<listitem>
+<para>Running under a debugger can cause an implicit
+<option>--nograb</option>, use <option>--dograb</option> to
+override.</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--sync</option>
+</term>
+<listitem>
+<para>Switches to synchronous mode for debugging.</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--fn</option>,<option>--font</option>
+<parameter>fontname</parameter>
+</term>
+<listitem>
+<para>Defines the application font.</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--bg</option>,<option>--background</option>
+<parameter>color</parameter>
+</term>
+<listitem>
+<para>Sets the default background color and an application palette
+(light and dark shades are calculated).</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--fg</option>,<option>--foreground</option>
+<parameter>color</parameter>
+</term>
+<listitem>
+<para>Sets the default foreground color</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--btn</option>,<option>--button</option>
+<parameter>color</parameter>
+</term>
+<listitem>
+<para>Sets the default button color.</para>
+</listitem>
+</varlistentry>
+
+
+<varlistentry>
+<term>
+<option>--name</option> <parameter>name</parameter>
+</term>
+<listitem>
+<para>Sets the application name.</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--title</option> <parameter>title</parameter>
+</term>
+<listitem>
+<para>Sets the application title (caption).</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--visual</option> <parameter>TrueColor</parameter>
+</term>
+<listitem>
+<para>Forces the application to use a <parameter>TrueColor</parameter>
+visual on an 8-bit display.</para>
+</listitem>
+</varlistentry>
+
+
+<varlistentry>
+<term>
+<option>--inputstyle</option> <parameter>inputstyle</parameter>
+</term>
+<listitem>
+<para>Sets XIM (X Input Method) input style.  Possible values are
+<parameter>onthespot</parameter>, <parameter>overthespot</parameter>,
+<parameter>offthespot</parameter> and
+<parameter>root</parameter>.</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--im</option> <parameter>XIM server</parameter>
+</term>
+<listitem>
+<para>Set XIM server.</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--noxim</option>
+</term>
+<listitem>
+<para>Disable XIM</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--reverse</option>
+</term>
+<listitem>
+<para>mirrors the whole layout of widgets</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
+<term>
+<option>--stylesheet</option> <replaceable>file.qss</replaceable>
+</term>
+<listitem>
+<para>applies the Qt stylesheet to the application widgets</para>
+</listitem>
+</varlistentry>
+
+</variablelist>
+
+</refsect1>
+
+
+<refsect1>
+<title>See Also</title>
+<para><replaceable>kdeoptions</replaceable>(7)</para>
+
+</refsect1>
+
+<refsect1>
+<title>Authors</title>
+<para>Man page written by &Lauri.Watts; &Lauri.Watts.mail;</para>
+</refsect1>
+
+</refentry>
--- /dev/null
+++ kdoctools-4.95.0/src/customization/xsl/all-l10n.xml
@@ -0,0 +1,80 @@
+<?xml version='1.0'?>
+<!DOCTYPE l:i18n SYSTEM "/usr/share/xml/docbook/stylesheet/docbook-xsl/common/l10n.dtd">
+<l:i18n xmlns:l="http://docbook.sourceforge.net/xmlns/l10n/1.0">
+<l:l10n language="af" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/af.xml"/>
+<l:l10n language="am" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/am.xml"/>
+<l:l10n language="ar" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/ar.xml"/>
+<l:l10n language="as" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/as.xml"/>
+<l:l10n language="ast" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/ast.xml"/>
+<l:l10n language="az" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/az.xml"/>
+<l:l10n language="bg" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/bg.xml"/>
+<l:l10n language="bn" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/bn.xml"/>
+<l:l10n language="bn_in" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/bn_in.xml"/>
+<l:l10n language="bs" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/bs.xml"/>
+<l:l10n language="ca" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/ca.xml"/>
+<l:l10n language="cs" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/cs.xml"/>
+<l:l10n language="cy" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/cy.xml"/>
+<l:l10n language="da" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/da.xml"/>
+<l:l10n language="de" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/de.xml"/>
+<l:l10n language="el" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/el.xml"/>
+<l:l10n language="en" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/en.xml"/>
+<l:l10n language="eo" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/eo.xml"/>
+<l:l10n language="es" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/es.xml"/>
+<l:l10n language="et" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/et.xml"/>
+<l:l10n language="eu" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/eu.xml"/>
+<l:l10n language="fa" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/fa.xml"/>
+<l:l10n language="fi" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/fi.xml"/>
+<l:l10n language="fr" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/fr.xml"/>
+<l:l10n language="ga" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/ga.xml"/>
+<l:l10n language="gl" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/gl.xml"/>
+<l:l10n language="gu" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/gu.xml"/>
+<l:l10n language="he" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/he.xml"/>
+<l:l10n language="hi" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/hi.xml"/>
+<l:l10n language="hr" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/hr.xml"/>
+<l:l10n language="hu" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/hu.xml"/>
+<l:l10n language="id" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/id.xml"/>
+<l:l10n language="is" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/is.xml"/>
+<l:l10n language="it" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/it.xml"/>
+<l:l10n language="ja" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/ja.xml"/>
+<l:l10n language="ka" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/ka.xml"/>
+<l:l10n language="kn" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/kn.xml"/>
+<l:l10n language="ko" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/ko.xml"/>
+<l:l10n language="ky" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/ky.xml"/>
+<l:l10n language="la" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/la.xml"/>
+<l:l10n language="lt" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/lt.xml"/>
+<l:l10n language="lv" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/lv.xml"/>
+<l:l10n language="ml" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/ml.xml"/>
+<l:l10n language="mn" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/mn.xml"/>
+<l:l10n language="mr" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/mr.xml"/>
+<l:l10n language="nb" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/nb.xml"/>
+<l:l10n language="nds" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/nds.xml"/>
+<l:l10n language="nl" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/nl.xml"/>
+<l:l10n language="nn" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/nn.xml"/>
+<l:l10n language="or" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/or.xml"/>
+<l:l10n language="pa" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/pa.xml"/>
+<l:l10n language="pl" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/pl.xml"/>
+<l:l10n language="pt" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/pt.xml"/>
+<l:l10n language="pt_br" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/pt_br.xml"/>
+<l:l10n language="ro" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/ro.xml"/>
+<l:l10n language="ru" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/ru.xml"/>
+<l:l10n language="sk" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/sk.xml"/>
+<l:l10n language="sl" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/sl.xml"/>
+<l:l10n language="sq" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/sq.xml"/>
+<l:l10n language="sr" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/sr.xml"/>
+<l:l10n language="sr_latn" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/sr_Latn.xml"/>
+<l:l10n language="sv" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/sv.xml"/>
+<l:l10n language="ta" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/ta.xml"/>
+<l:l10n language="te" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/te.xml"/>
+<l:l10n language="th" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/th.xml"/>
+<l:l10n language="tl" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/tl.xml"/>
+<l:l10n language="tr" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/tr.xml"/>
+<l:l10n language="uk" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/uk.xml"/>
+<l:l10n language="vi" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/vi.xml"/>
+<l:l10n language="xh" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/xh.xml"/>
+<l:l10n language="zh" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/zh.xml"/>
+<l:l10n language="zh_cn" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/zh_cn.xml"/>
+<l:l10n language="zh_tw" href="/usr/share/xml/docbook/stylesheet/docbook-xsl/common/zh_tw.xml"/>
+<l:l10n language="sr_ijekavian" href="sr_ijekavian.xml"/>
+<l:l10n language="sr_ijekavianlatin" href="sr_ijekavianlatin.xml"/>
+<l:l10n language="sr_latin" href="sr_latin.xml"/>
+</l:i18n>
